packit_service/worker/database.py

```diff
@@ -0,0 +1,23 @@
+from datetime import timedelta, datetime
+from logging import getLogger
+from os import getenv
+
+from packit_service.constants import SRPMBUILDS_OUTDATED_AFTER_DAYS
+from packit_service.models import SRPMBuildModel
+
+logger = getLogger(__name__)
+
+
+def discard_old_srpm_build_logs():
+    """Called periodically (see celery_config.py) to discard logs of old SRPM builds."""
+    logger.debug("About to discard old SRPM build logs.")
+    outdated_after_days = getenv(
+        "SRPMBUILDS_OUTDATED_AFTER_DAYS", SRPMBUILDS_OUTDATED_AFTER_DAYS
+    )
+    ago = timedelta(days=outdated_after_days)
+    for build in SRPMBuildModel.get_older_than(ago):
+        age = datetime.utcnow() - build.build_submitted_time
+        logger.debug(
+            f"SRPM build {build.id}, age '{age}' is older than '{ago}'. Discarding logs."
+        )
+        build.set_logs("")
```

Comment:
How is the value represented in the database? (`None` vs `""` vs `"log removed"`)

Reply:
Not sure I understand the question. [SRPMBuildModel.logs is a Text type](https://github.com/packit/packit-service/blob/c79d591709281503c11ae199c91598d3f471bcdf/packit_service/models.py#L1195) so I guess ""  is represented as an empty string in the database. I don't think we can put `None` there and if you're suggesting to put some explanation (e.g. "Logs have been removed") in place of the original logs, well, that's certainly doable.

Reply:
Yes, I was thinking about something like that. (But am still not sure how the text is represented -- e.g. if  `""` and `"not available"` differs in disk space.)

If the explanation is doable, that would be nice. Or, use an empty string and show `Logs not available` in the dashboard (probably easier to do and saves some space).

Reply:
Oh, you've already done that..;) Thanks

Reply:
I completely forgot about the `NULL` value, which is what we want, I think.
I've added a commit, which sets `logs` and `url` to `NULL` for the outdated SRPM builds.
And we'll show some message in the dashboard if the log is missing (it already handles missing/empty `url`).

I'll squash those commits later if you agree it's better than storing empty string and/or "not available".